tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -424,39 +436,40 @@ def test_bootstrap(self, monkeypatch, test_repo):
             "last_update": fake_time,
             "status": "Task finished.",
         }
+        assert fake_datetime.now.calls == [pretend.call()]
         assert repository.Metadata.from_dict.calls == [
-            pretend.call({"md_k1": "md_v1"}),
-            pretend.call({"md_k2": "md_v2"}),
+            pretend.call(payload["metadata"]["root"])
         ]
-        assert fake_datetime.now.calls == [pretend.call()]
-        assert test_repo._persist.calls == [
-            pretend.call("fake_metadata", "1.root"),
-            pretend.call("fake_metadata", "1.snapshot"),
+        assert repository.Key.from_securesystemslib_key.calls == [
+            pretend.call({"k": "v"}),
+            pretend.call({"k": "v"}),
+        ]
+        assert test_repo._key_storage_backend.get.calls == [
+            pretend.call("online_public_key"),
+            pretend.call("online_public_key"),
         ]
```

Comment:
Could please add asserts on the calls for `test_repo._bump_expiry`, `test_repo._sign` and `test_repo._persist`?